tower-buffer/src/error.rs

```diff
@@ -85,11 +85,12 @@ where
     }
 }
 
-impl<T> std::error::Error for SpawnError<T>
-where
-    T: std::error::Error + 'static,
+impl<T: std::fmt::Debug> std::error::Error for SpawnError<T>
+// where
+//     T: std::error::Error + 'static,
```

Comment:
Had trouble with this since `Connection` doesn't impl `std::error::Error`

cc @carllerche

Reply:
I'm not following this. I forgot the context.

Reply:
I accidentally pushed to master... but would this help? https://github.com/tower-rs/tower/commit/733b597f38a1acf1c9a25ed4528a96d168bb4cd8

Reply:
Yup, this should fix my issue, ill bring this branch current to master later today. Thanks.